library/src/main/java/com/vinted/coper/CoperImpl.kt

```diff
@@ -46,6 +58,7 @@ internal class CoperImpl(private val fragmentManager: FragmentManager) : Coper {
     }
 
     @Synchronized
+    @Deprecated("Use getFragmentSafely")
```

Comment:
Why are deprecating internal stuff? I suppose you should prefer the usage of a safer function.

Reply:
I added it for myself and other contribution, that it would not be used